app/services/send_reference_request_reminder_to_teacher.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+#
+class SendReferenceRequestReminderToTeacher
```

Comment:
Rather than introducing a new service class and job for sending these emails, can we instead send the email at the same point that we send the reminder to the referees?

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/13d4dd3201ad687d64ed5e19ab41f6269acb952a/app/models/reference_request.rb#L101-L103

Reply:
Talked in person and we need this to send one email together.